wwwroot/buy-anything.html

```diff
@@ -0,0 +1,414 @@
+                                </label>
+                                <div class="card-content">
+                                    <p>Orders made through Wasabi include an affiliate code and a fair cut is given to zkSNACKs, similar to standard affiliation programs.</p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq10">
+                                <label for="faq10">
+                                    10. Will I get in trouble with tax authorities because of this?
+                                </label>
+                                <div class="card-content">
+                                    <p>We can't help you with tax issues, you need to know the laws of your own country. In general, if you haven't paid taxes, you may get in trouble for not paying the taxes, not for spending your money with Shopinbit’s service.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq11">
+                                <label for="faq11">
+                                    11. What countries are not allowed?
+                                </label>
+                                <div class="card-content">
+                                    <p>Iran, North Korea are not allowed. Ukraine is currently halted due to the ongoing war.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq12">
+                                <label for="faq12">
+                                    12. Does the service include shipping to remote areas? What shipping service will be used?
+                                </label>
+                                <div class="card-content">
+                                    <p>Shopinbit ships anywhere, if you can pay for it. The service depends on the country. Their team has experience with difficult shipping conditions.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq13">
+                                <label for="faq13">
+                                    13. What is the return and reimbursement policy?
+                                </label>
+                                <div class="card-content">
+                                    <p>After you receive your order, you have a 14-day right of return. This means that if you do not like it, you can send it back (you will have to pay for the shipping).</p>
```

Comment:
I'd mention something like: please refer to Shopinbit's Terms and Conditions for up-to-date conditions.
We cannot be sure it is always up to date/Shopinbit didn't make any changes in the meantime

Reply:
14 day return policy is EU law, not their choice, so will not change. And if it will, we will notice.